Add documentation on using a minimal config file (#7158)
PyCQA/pylint: doc/user_guide/configuration/index.rst

    pylint --disable=bare-except,invalid-name --class-rgx='[A-Z][a-z]+' --generate-toml-config

> In practice, you should create a minimal configuration file which contains only
> configuration overrides. For all other options, Pylint will use default values.

Reviewer: ```suggestion
In practice, it is often better to create a minimal configuration file which only contains
configuration overrides. For all other options, Pylint will use its default values.
```
